src/compiler/compile/nodes/Element.ts

```diff
@@ -555,6 +555,69 @@ export default class Element extends Node {
 				});
 			}
 		}
+
+		if (handlers_map.has('click')) {
+
+			const a11y_interactive = new Set([
+				'a',
+				'button',
+				'select'
+			]);
+
+			if (a11y_interactive.has(this.name)) {
+				return;
+			}
+
+			if (this.name === 'input') {
+				const input_type = attribute_map.get('type');
+				const input_type_interactive = new Set(['button', 'checkbox', 'color', 'file', 'image', 'radio', 'reset', 'submit']);
```

Comment:
Could you move this out to the top level?

Reply:
Done